packages/core/src/sdk/search/useSuggestions.ts

```diff
@@ -74,6 +75,7 @@ function useSuggestions(term: string, limit: number = MAX_SUGGESTIONS) {
   })
 
   return {
+    data,
     terms: (data?.search.suggestions.terms ?? []).slice(0, limit),
     products: (data?.search.suggestions.products ?? []).slice(0, limit),
```

Comment:
does still makes sense to return these when we're already returning data?

Reply:
I was thinking about that and [change it here](https://github.com/vtex/faststore/pull/2051/commits/db48b37e20f78a0e5fd21e9165b5bade6720d9c2).